panther_cli/main.py

```diff
@@ -294,12 +349,25 @@ def setup_parser() -> argparse.ArgumentParser:
                             required=True)
     zip_parser.set_defaults(func=zip_policies)
 
+    upload_parser = subparsers.add_parser(
+        'upload', help='Upload specified policies to a Panther deployment.')
+    upload_parser.add_argument('--policies',
+                               type=str,
+                               help='The relative path to Panther policies.',
+                               required=True)
+    upload_parser.add_argument('--output-path',
+                               default='.',
+                               type=str,
+                               help='The relative path to Panther policies.',
+                               required=False)
+    upload_parser.set_defaults(func=upload_policies)
+
     return parser
 
 
 def run() -> None:
     logging.basicConfig(format='[%(levelname)s]: %(message)s',
-                        level=logging.DEBUG)
+                        level=logging.INFO)
```

Comment:
boto3 spits out a lot of DEBUG level output which we catch with this set to DEBUG. I figure it makes sense to default to INFO anyways for most users.